src/routes/account.ts

```diff
@@ -99,6 +103,37 @@ router.get(
   }),
 );
 
+// router.get(
+//   '/init',
+//   catchErrors(async (req, res, next) => {
+//     return res.render('initAccount',);
+//   }),
+// );
+
+// router.post(
+//   '/init',
+//   catchErrors(async (req, res, next) => {
+//     const { error, value } = jf.validateAsClass(req.query, controller.ResetCredentials);
+
+//     if (error) {
+//       throw new HttpException(400, { message: `Invalid request: ${error.message}` });
+//     }
+
+//     // I have no idea why joiful throws a fit here but this is the necessary workaround
+//     const verifyEmail = await controller.verifyEmail(
+//       value as unknown as controller.ResetCredentials,
+//     );
+
+//     return res.render('initAccountSuccess', {
+//       id: value.id,
+//       key: value.key,
+//       email: verifyEmail.email,
+//     });
+//   }),
+// );
+
+
+
```

Comment:
Not sure why the commented block here but of course let me know if you need any help w/this